website/views.py

```diff
@@ -24,7 +23,7 @@
 from website.institutions.views import serialize_institution
 
 from osf.models import BaseFileNode, Guid, Institution, PreprintService, AbstractNode, Node
-from website.settings import EXTERNAL_EMBER_APPS, PROXY_EMBER_APPS, INSTITUTION_DISPLAY_NODE_THRESHOLD, DOMAIN
```

Comment:
Should this also be deleted from settings?

Reply:
yeah I think so!